### PR TITLE
Introduce `cometbtf` command to replace `tendermint`

### LINKED ISSUE #10000: Alias `tendermint` sub-command as `cometbft`

### PR DESCRIPTION
**Pull request by ValentinTrinque**
Close #10000

### CODE REVIEW

File: cmd/vega/commands/root.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func Main(ctx context.Context) error {
 	if len(os.Args) >= 2 {
 		switch os.Args[1] {
 		case "tendermint", "tm":
```
Comment (jeremyletang) [inline cmd/vega/commands/root.go:50]: can you add as well "cometbft" in here.

Comment (ValentinTrinque) [reply cmd/vega/commands/root.go:50]: nice catch.

Comment (ValentinTrinque) [reply cmd/vega/commands/root.go:50]: I simply ran `vega cometbft`.

File: cmd/vega/commands/cometbft.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+		cmtcmd.ShowNodeIDCmd,
+		cmtcmd.GenNodeKeyCmd,
+		cmtcmd.VersionCmd,
+		cmtcmd.RollbackStateCmd,
+		cmtcmd.CompactGoLevelDBCmd,
+		cmtdebug.DebugCmd,
+		cmtcli.NewCompletionCmd(rootCmd, true),
+	)
+
+	baseCmd := cmtcli.PrepareBaseCmd(rootCmd, "TM", os.ExpandEnv(filepath.Join("$HOME", cmtcfg.DefaultTendermintDir)))
```
Comment (jeremyletang) [inline cmd/vega/commands/cometbft.go:88]: That env thing have changed with cometbft, I guess we need to support also the COMET dir or whatever they renamed it with comet

Comment (ValentinTrinque) [reply cmd/vega/commands/cometbft.go:88]: Ah right ! I looked into it already, and I apparently should have not trusted the outdated comment inside the function that still talk about `TM`. Indeed, their main is using `CMT`.

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**